packages/core/src/fhir-to-cda/composition-splitter.ts

```diff
@@ -26,13 +27,20 @@ export function splitBundleByCompositions(fhirBundle: Bundle): Bundle[] {
       ? findResourceInBundle(fhirBundle, patientReference)
       : undefined;
     if (!patientResource) {
-      throw new BadRequestError("Patient resource not found");
+      const msg = `Invalid Patient reference in the subject field`;
+      capture.error(msg, {
+        extra: {
+          patientReference,
+          organizationReference,
+        },
+      });
```

Comment:
We prob should add more context here as to which use case/feature is this related to.

I wonder if we should start adding the feature/use case as a short prefix to the title, something like `[contrib] Invalid Patient reference in the subject field` - likely something to take to #engineering, though.